### DISCUSSION TITLE
Option to disable "Accept" popup for incoming connection

### DISCUSSION THREAD

**Discussion by ThiefMaster** [rustdesk/rustdesk#8391] [General]
I exclusively use tools like RustDesk to access my own computer remotely, I never plan to receive an incoming connection while I'm on the computer myself.

So the fact that the "Accept" dialog exists and cannot be disabled is a security risk for me: It means I just need to do one unlucky click while there's a malicious incoming connection (well, to be fair, someone would probably need to know my ID for that) and it's game over. My strong password won't help, 2FA won't help. Because none of those are needed when clicking the "Accept" button.

**Comment by rustdesk** [comment]
![image](https://github.com/rustdesk/rustdesk/assets/71636191/a321961e-6945-468f-a67b-8df32c483f4d)

**Reply by ThiefMaster** [reply]
Oh indeed, I missed it because of the very weird/flaky/inconsistent scrolling behavior of that window...
